types/react-native-sortable-grid/index.d.ts

```diff
@@ -115,7 +115,7 @@ interface SortableGridProps {
      * containing a key transform, which is an array of transformations.
      */
     dragStartAnimation?: {
-        transform: ReadonlyArray<{ [type: string]: Animated.AnimatedInterpolation }>;
+        transform: ReadonlyArray<{ [type: string]: Animated.AnimatedInterpolation<number|string> }>;
```

Comment:
Is this due to changes in 0.70 types? This is fine if this library supports 0.70. Otherwise it would probably be safer to use 0.69 types in this package by adding the prop paths alias (like https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59443/files did).

Reply:
Yes It's due to changes in 0.70. I was following the changes listed here: https://github.com/facebook/react-native/compare/0.69-stable...0.70-stable#diff-fa3321d36f6dd6ffef6921d43504475e7af4f7bec90bbbacd833240223584ceb

Reply:
@eps1lon Is it a matter of specifying the version here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native-sortable-grid/tsconfig.json#L12 ?